### ISSUE TITLE
Custom kong plugins are not displayed in kong manager with the list of plugins

### ISSUE THREAD

**Issue by medbayoudh** [Kong/kong#12740]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Kong version (`$ kong version`)

3.6.1

### Current Behavior

we have a bunch of custom plugin for our kong gateway. We were using `3.4.2` and we were able to see those custom plugins in the list of plugins in `kong-manager`. Now after upgrading to `3.6.1` we are not able to see them. Configuring routes with custom plugins using the REST API is working fine and the plugins are recognized but unfortunately we are not able to configure them directly from `kong-manager` (which is a way easier)

### Expected Behavior

- Custom plugins are displayed in the list of plugins 
- Configuring custom plugins for routes or services using `kong-manager`

### Steps To Reproduce

1. install custom plugin P
2. create route A
3. configure custom plugin P for route A using kong-manager
4. custom-plugin does not appear in the list of configurable plugins

**Comment by Maurici093** [comment]
I have exactly the same behavior today :'(

**Comment by sumimakito** [comment]
Hello @medbayoudh - Thank you for reporting this. We are currently looking into this issue. 

This issue should only affect the plugin select page. Meanwhile, you may access the URL for the plugin configuration page directly as a workaround. 

For example: 

```
http://localhost:8080/default/plugins/select/<YOUR_CUSTOM_PLUGIN_NAME>
```

**Comment by sumimakito** [comment]
Hi @medbayoudh - I just confirmed with the team that this issue has been fixed in the latest as well as the 3.6.x branch. This fix will be delivered in the next major or minor release. I will close this issue for now. You can reply or reopen it if you have any other needs.

**Comment by pmeakins** [comment]
Issue has reappeared in 3.9.0.0, my pluging in 3.8.0.0 is working and visible.    In 3.9.0.0, the plugin works but not visible via the 'New Plugin' screen, I can make the call to the default/plugins/select/<plugin> and that works fine.   Even creating a new pluging and thats visible in the route screen,   on adding a second,  still the plugin is only available via the direct url.
